bin/rose-suite-run

```diff
@@ -117,11 +117,14 @@
 #         Each `KEY` in this space delimited list switches on an optional
 #         configuration. The configurations are applied first-to-last.
 #
-# JINJA2 VARIABLES
-#     `rose suite-run` provides the following Jinja2 variables to the suite.
+# SUITE VARIABLES
+#     `rose suite-run` provides the following variables to the suite.
```

Comment:
We should make it clear that these variables are template variables not environment variables.

```suggestion
#     `rose suite-run` provides the following variables to the suite as template variables (i.e. Jinja2 or EmPy).
```

*Why this matters: `$ROSE_ORIG_HOST` is actually an environment variable but isn't only exported locally so we advise people use `{{ ROSE_ORIG_HOST }}`.*